docs/core-services/position-service.md

```diff
@@ -21,11 +21,19 @@ This service provides the following configuration parameters:
 - **provider** - species which position provider use, can be gpsd or serial. 
```

Comment:
- **provider** - speci**fi**es which position provider use, can be gpsd, serial or modemManager.